controller/src/main/java/io/pravega/controller/store/stream/tables/State.java

```diff
@@ -9,12 +9,39 @@
  */
 package io.pravega.controller.store.stream.tables;
 
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
 /**
```

Comment:
I understand this is only a temporary class, until the stream creation and stream config update operations are remodeled as Event Processors. Is this correct? In this PR State acts as a lock that prevents stream config update from running concurrently with a scale operation, right?